iFixit/react-commerce
# Add available facets as filters

This PR introduces the business logic to display all facets accessible from the Algolia index. This will be useful for @sterlinghirsh to debug the Algolia index and see if all necessary facets are visible in the collection.

This PR does **NOT** contain: 
- Facet specific implementations (like the price slider), the UI should be considered a draft
- Localization of facets

## QA

1. Visit the Vercel preview
2. Visit a few collections that contain products and verify that facets are dynamically populated based on the collection

> ⚠️ Count badges next to facet values are displayed for debug purposes and might display wrong values. When the filtering behavior will be throughly debugged these badges will be removed.
